recipe/0002-remove-openmpi-lmpi_cxx.patch

```diff
@@ -0,0 +1,15 @@
+*** configure.ac	2019-01-06 16:51:46.000000000 -0700
+--- configure.fix	2020-01-24 11:32:38.334285241 -0700
+***************
+*** 605,614 ****
+  fi
+  FATHOM_CHECK_MPITYPE
+  
+- if (test "$MPIFAMILY" = "OPENMPI" && test "x$ENABLE_FORTRAN" != "xno"); then
+-   FCLIBS="$FCLIBS -lmpi_cxx"
+- fi
+- 
```

Comment:
@vijaysm, I'd suggest removing these lines in the current version of MOAB so we won't need this patch in the future.  Or maybe figure out which older versions of Open MPI need this and include a version check in this `if`.

Reply:
Good catch @xylar . I didn't realize we had a special addition for OpenMPI. @iulian787 can we test a MOAB build with OpenMPI with the above change ? I do not remember why we need the MPI C++ libs here for linkage.